ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -356,6 +366,107 @@ class NavigationBindings {
         circle: ['labelOptions']
     };
 
+    // Define types for editable fields per annotation
+    public static annotationsFieldsTypes: Record<string, string> = {
+        addButton: 'string',
+        algorithm: 'string',
+        arrowInfinityLine: 'string',
+        arrowRay: 'string',
+        arrowSegment: 'string',
+        average: 'string',
```

Comment:
I think they are here, check `algorithm` and `average`, but at the same point, I'm not sure if that is correct.
Meaby I'm not aware of some cases where those properties should be displayed in input instead of a dropdown.
But from what I remember, this should not be such a case.. @pawelfus 

So I'm not quite sure if we should mix dropdown properties with inputs. I personally would separate them.
@hubertkozik if you also decide so, please check these two properties if they should be here. From what I can see everything works as expected and you can easily change the `average` method in the `Disparity index` indicator.

Reply:
No, algorithm option (there's only one at this moment) is always a dropdown. 

Good point, let's not mix field type with field possible values. Thanks!